pytest_html/plugin.py

```diff
@@ -41,9 +41,11 @@ def pytest_addoption(parser):
     group.addoption('--html', action='store', dest='htmlpath',
                     metavar='path', default=None,
                     help='create html report file at given path.')
-    group.addoption('--self-contained-html', action="store", dest='self_contained_html', default=False,
-        help='If it is true, asserts that the HTML will not ' +
-             'depend of external files, risk of breaking CSP. DEFAULT: False')
+    group.addoption('--self-contained-html', action="store",
```

Comment:
I think `action` should be `store_true` so the value is `True` when this option is provided on the command line. Also, please use single quotes wherever possible.